addons/web-support/api/src/main/java/org/jboss/windup/web/addons/websupport/rest/graph/applicationDetails/HintReducedDTO.java

```diff
@@ -0,0 +1,60 @@
+package org.jboss.windup.web.addons.websupport.rest.graph.applicationDetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.enterprise.inject.Vetoed;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ */
+@Vetoed
+@JsonIgnoreProperties({"handler", "delegate"})
+public class HintReducedDTO
+{
+    private int title;
+    private int javaFQCN;
```

Comment:
Yes, this is the qualified name of the type reference. It is using the string cache, as it is very common for these to have thousands of references to the same name.